Links.html

```diff
@@ -9,36 +9,37 @@
         #useful-links a {color: #8E443D;}
         </style>
 </head>
-<ul>
-    <li><a href="index.html">Home</a></li>
-    <li><a href="Glossary.html">Glossary</a></li>
-    <li><a href="Links.html">Links</a></li></ul>
-    <title>Alison's CSD120 Website</title>
-    <meta charset="utf-8">
-</head>
 <body>
+    <header>
+        <nav>
+            <ul>
+                <li><a href="index.html">Home</a></li>
+                <li><a href="Glossary.html">Glossary</a></li>
+                <li><a href="Links.html">Links</a></li>
+            </ul>
+        </nav>
+    </header>
 
 
 
     <main>
-
-        <h1>Useful Links</h1>
+        <section id="useful-links">  
+            <h1>Useful Links</h1>
 
            <ul>
-         <section id="useful-links">  
                 <li><a href="https://validator.w3.org/nu/" target="_blank">HTML Validator</a></li>
                     <li><a href="https://jigsaw.w3.org/css-validator/" target="_blank">CSS Validator</a></li>
-                    <li><a href="https://developer.mozilla.org/en-US" target="_black">MDN</a></li>
+                    <li><a href="https://developer.mozilla.org/en-US" target="_blank">MDN</a></li>
                 <li><a href="https://caniuse.com" target="_blank">caniuse.com</a></li>
-          </section> 
            </ul>
+        </section> 
     </main>
 
     <footer>
-        <hr></hr>
+        <hr>
         <blockquote>You can make anything by writng
-        <br><b>&mdash; C.S. Lewis</b></blockquote>
-        <p id="page-copyright">&copy; 2019 <a href="mailto:Alison.Nadjiwon@student.saultcollege.ca?subjext=Message from your CSD120 website"> Alison Nadjiwon</a></p>
+        <br><span class="byline"><b>&mdash; C.S. Lewis</b></span></blockquote>
+        <p id="page-copyright">&copy; 2019 <a href="mailto:Alison.Nadjiwon@student.saultcollege.ca?subjext=Message%20from%20your%20CSD120%20website"> Alison Nadjiwon</a></p>
```

Comment:
Missing byline class
Missing URL-encoded spaces (%20) — I'm guessing this is another error in merging